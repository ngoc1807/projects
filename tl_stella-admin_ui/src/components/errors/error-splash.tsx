import { FC } from "react";
import {
  Box,
  Button,
  Typography,
  useMediaQuery,
  useTheme,
} from "@mui/material";
import { StellaIcon } from "../icons/stella-icon";
import Link from "next/link";

interface Props {
  errorMessage?: string;
  authError?: boolean;
}

export const ErrorSplash: FC<Props> = ({ errorMessage, authError = false }) => {
  const theme = useTheme();
  const breakpointSmallAndUp = useMediaQuery(theme.breakpoints.up("sm"));
  return (
    <>
      <Box
        sx={{
          marginTop: "10%",
        }}
      >
        <Box
          sx={{
            marginLeft: "-8px",
          }}
        >
          <StellaIcon
            variant="long"
            size={24}
            color1={theme.palette.secondary.main}
            color2={theme.palette.common.white}
          />
        </Box>
        <Typography
          variant="h1"
          component="h1"
          sx={{
            color: "common.white",
            marginTop: 5,
            marginBottom: 7,
            maxWidth: "50%",
            minWidth: 400,
          }}
        >
          Ohh, No!
        </Typography>
        <Typography
          variant="h5"
          component="h2"
          color="common.white"
          sx={{
            marginBottom: 2,
          }}
        >
          There has been an error.
        </Typography>
        {errorMessage !== undefined && (
          <Typography color="common.white" variant="subtitle1">
            {errorMessage}
          </Typography>
        )}
      </Box>
      {authError ? (
        <>
          <Box
            sx={{
              marginTop: 8,
              marginBottom: 2,
            }}
          >
            <Button variant="outlined" href="/">
              Login
            </Button>
            <Button
              variant="contained"
              color="secondary"
              sx={{
                ml: breakpointSmallAndUp ? 1 : 0,
                mt: breakpointSmallAndUp ? 0 : 1,
              }}
              href="/logout"
            >
              Logout
            </Button>
          </Box>
          <Box
            sx={{
              a: { color: "inherit", textDecoration: "none" },
            }}
          >
            <Link href="/">Home &#62;</Link>
          </Box>
        </>
      ) : (
        <Button
          variant="outlined"
          href="/"
          sx={{
            marginTop: 8,
            marginBottom: 2,
          }}
        >
          Home
        </Button>
      )}
    </>
  );
};
