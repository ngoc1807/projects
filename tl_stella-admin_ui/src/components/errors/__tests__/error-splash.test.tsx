import { cleanup, render } from "@testing-library/react";
import { ErrorSplash } from "../error-splash";

describe("error-splash", () => {
  afterEach(() => {
    cleanup();
  });
  it("should render correctly when no error message is passed", () => {
    const { container } = render(<ErrorSplash />);
    expect(container.firstChild).toMatchSnapshot();
  });
  it("should render correctly when an error message is passed", () => {
    const errorMessage = "404 Error: Page Not Found";
    const { container } = render(<ErrorSplash errorMessage={errorMessage} />);
    expect(container.firstChild).toMatchSnapshot();
  });
  it("should render correctly when error is an auth error", () => {
    const errorMessage = "Authentication Error";
    const { container } = render(
      <ErrorSplash authError={true} errorMessage={errorMessage} />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
