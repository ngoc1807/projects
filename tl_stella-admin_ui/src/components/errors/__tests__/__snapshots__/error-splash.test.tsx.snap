// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error-splash should render correctly when an error message is passed 1`] = `
<div
  class="MuiBox-root css-ui1dqj"
>
  <div
    class="MuiBox-root css-1aisqbv"
  >
    <svg
      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-3g5fjw-MuiSvgIcon-root"
      focusable="false"
      role="img"
      viewBox="0 0 1317 105"
    >
      <path
        d="M583.82 100C580.54 99.9764 577.4 98.6621 575.081 96.3415C572.763 94.0208 571.451 90.8805 571.43 87.6V35.78H583.2V85C583.2 86.67 583.3 87.61 583.51 87.81C583.72 88.01 584.65 88.13 586.32 88.13H620.66C621.592 88.2346 622.536 88.1251 623.42 87.81C623.66 87.61 623.78 86.67 623.78 85V35.78H635.64V87.6C635.614 90.8806 634.299 94.0194 631.979 96.3392C629.659 98.659 626.521 99.9739 623.24 100H583.82Z"
        fill="#9c27b0"
      />
      <path
        d="M669.53 100V47.64H643.31V35.78H707.52V47.64H681.39V100H669.53Z"
        fill="#9c27b0"
      />
      <path
        d="M727.5 100C724.22 99.9764 721.08 98.6621 718.761 96.3415C716.443 94.0208 715.131 90.8805 715.11 87.6V48.18C715.131 44.8995 716.443 41.7592 718.761 39.4385C721.08 37.1179 724.22 35.8036 727.5 35.78H766.92C770.201 35.8061 773.339 37.121 775.659 39.4408C777.979 41.7606 779.294 44.8994 779.32 48.18V87.6C779.294 90.8806 777.979 94.0194 775.659 96.3392C773.339 98.659 770.201 99.9739 766.92 100H727.5ZM730 88.14H764.34C765.272 88.2446 766.216 88.1351 767.1 87.82C767.34 87.62 767.46 86.68 767.46 85.01V50.77C767.46 49.1 767.34 48.17 767.1 47.96C766.216 47.6449 765.272 47.5355 764.34 47.64H730C728.33 47.64 727.4 47.75 727.19 47.96C726.98 48.17 726.88 49.1 726.88 50.77V85C726.88 86.67 726.98 87.61 727.19 87.81C727.4 88.01 728.33 88.14 730 88.14Z"
        fill="#9c27b0"
      />
      <path
        d="M790.2 100V35.78H803.13L826.41 63.52L849.6 35.78H862.6V100H850.74V52.91L826.39 81.91L802 53V100H790.2Z"
        fill="#9c27b0"
      />
      <path
        d="M886.25 100C882.97 99.9764 879.83 98.6621 877.511 96.3415C875.193 94.0208 873.881 90.8805 873.86 87.6V48.18C873.881 44.8995 875.193 41.7592 877.511 39.4385C879.83 37.1179 882.97 35.8036 886.25 35.78H925.68C928.961 35.8061 932.099 37.121 934.419 39.4408C936.739 41.7606 938.054 44.8994 938.08 48.18V87.6C938.054 90.8806 936.739 94.0194 934.419 96.3392C932.099 98.659 928.961 99.9739 925.68 100H886.25ZM888.75 88.14H923.09C924.026 88.243 924.972 88.1336 925.86 87.82C926.1 87.62 926.21 86.68 926.21 85.01V50.77C926.21 49.1 926.1 48.17 925.86 47.96C924.972 47.6464 924.026 47.5371 923.09 47.64H888.75C887.09 47.64 886.15 47.75 885.94 47.96C885.73 48.17 885.63 49.1 885.63 50.77V85C885.63 86.67 885.73 87.61 885.94 87.81C886.15 88.01 887.09 88.14 888.75 88.14Z"
        fill="#9c27b0"
      />
      <path
        d="M972 100V47.64H945.78V35.78H1010V47.64H983.87V100H972Z"
        fill="#9c27b0"
      />
      <path
        d="M1017.36 100V35.78H1029.05V100H1017.36Z"
        fill="#9c27b0"
      />
      <path
        d="M1304.36 100V35.78H1316.05V100H1304.36Z"
        fill="#9c27b0"
      />
      <path
        d="M567.43 100H553.7L527.65 54.64L501.43 100H487.78L527.65 31.28L567.43 100Z"
        fill="#9c27b0"
      />
      <path
        d="M1297.43 100H1283.7L1257.7 54.64L1231.43 100H1217.78L1257.65 31.28L1297.43 100Z"
        fill="#9c27b0"
      />
      <path
        d="M1035.81 35.78H1049.54L1075.54 81.14L1101.76 35.78H1115.41L1075.54 104.5L1035.81 35.78Z"
        fill="#9c27b0"
      />
      <path
        d="M1122.24 100V35.78H1181.37V47.64H1134.1V61.92H1172.1V73.87H1134.1V88.14H1181.37V100H1122.24Z"
        fill="#9c27b0"
      />
      <path
        d="M53.87 44.86H101.03V36.1L53.87 36.21C49.0305 36.2179 44.3917 38.1446 40.9706 41.5676C37.5494 44.9906 35.6253 49.6305 35.62 54.47V68.13H83C85.4896 68.1327 87.8764 69.1228 89.6368 70.8832C91.3972 72.6436 92.3874 75.0304 92.39 77.52V91.35H35.62V100H101V77.5C100.992 72.7221 99.0954 68.141 95.7235 64.756C92.3515 61.3709 87.7779 59.4564 83 59.43H44.29V54.43C44.2979 51.8926 45.3101 49.4615 47.1053 47.6682C48.9005 45.8749 51.3326 44.8653 53.87 44.86V44.86Z"
        fill="#fff"
      />
      <path
        d="M109.76 36.1V44.84H137.28V100H146.08V44.84H173.52V36.1H109.76Z"
        fill="#fff"
      />
      <path
        d="M258.13 91.2V36.16H249.33V100H313.09V91.2H258.13Z"
        fill="#fff"
      />
      <path
        d="M418.56 31.36L384.1 91.2H330.62V36.16H321.82V100H379.03H385.58H389.19L418.6 48.9L448.32 100.09H458.49L418.56 31.36Z"
        fill="#fff"
      />
      <path
        d="M240.59 44.84V36.1H182.26V100H240.59V91.2H191.06V68H221.76V59.3H191.06V44.84H240.59Z"
        fill="#fff"
      />
      <path
        d="M26.79 0.5C26.79 7.60976 23.9691 14.429 18.9464 19.4611C13.9238 24.4931 7.10974 27.3268 0 27.34C7.11841 27.34 13.9453 30.1678 18.9787 35.2013C24.0122 40.2347 26.84 47.0616 26.84 54.18C26.84 47.0616 29.6678 40.2347 34.7013 35.2013C39.7347 30.1678 46.5616 27.34 53.68 27.34C46.5616 27.34 39.7347 24.5122 34.7013 19.4788C29.6678 14.4453 26.84 7.61841 26.84 0.5H26.79Z"
        fill="#9c27b0"
      />
      <path
        d="M438.63 47.28V38.49H433.57V36.1H445.65V38.49H441.16V47.28H438.63Z"
        fill="#fff"
      />
      <path
        d="M445.61 47.28V36.1H449.26L451.97 40.73L454.66 36.1H458.3V47.28H455.92V38L452 44.8L448 38V47.24L445.61 47.28Z"
        fill="#fff"
      />
      <title>
        STELLA Automotive AI
      </title>
    </svg>
  </div>
  <h1
    class="MuiTypography-root MuiTypography-h1 css-1udmwja-MuiTypography-root"
  >
    Ohh, No!
  </h1>
  <h2
    class="MuiTypography-root MuiTypography-h5 css-10hv533-MuiTypography-root"
  >
    There has been an error.
  </h2>
  <h6
    class="MuiTypography-root MuiTypography-subtitle1 css-as4pw1-MuiTypography-root"
  >
    404 Error: Page Not Found
  </h6>
</div>
`;

exports[`error-splash should render correctly when error is an auth error 1`] = `
<div
  class="MuiBox-root css-ui1dqj"
>
  <div
    class="MuiBox-root css-1aisqbv"
  >
    <svg
      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-3g5fjw-MuiSvgIcon-root"
      focusable="false"
      role="img"
      viewBox="0 0 1317 105"
    >
      <path
        d="M583.82 100C580.54 99.9764 577.4 98.6621 575.081 96.3415C572.763 94.0208 571.451 90.8805 571.43 87.6V35.78H583.2V85C583.2 86.67 583.3 87.61 583.51 87.81C583.72 88.01 584.65 88.13 586.32 88.13H620.66C621.592 88.2346 622.536 88.1251 623.42 87.81C623.66 87.61 623.78 86.67 623.78 85V35.78H635.64V87.6C635.614 90.8806 634.299 94.0194 631.979 96.3392C629.659 98.659 626.521 99.9739 623.24 100H583.82Z"
        fill="#9c27b0"
      />
      <path
        d="M669.53 100V47.64H643.31V35.78H707.52V47.64H681.39V100H669.53Z"
        fill="#9c27b0"
      />
      <path
        d="M727.5 100C724.22 99.9764 721.08 98.6621 718.761 96.3415C716.443 94.0208 715.131 90.8805 715.11 87.6V48.18C715.131 44.8995 716.443 41.7592 718.761 39.4385C721.08 37.1179 724.22 35.8036 727.5 35.78H766.92C770.201 35.8061 773.339 37.121 775.659 39.4408C777.979 41.7606 779.294 44.8994 779.32 48.18V87.6C779.294 90.8806 777.979 94.0194 775.659 96.3392C773.339 98.659 770.201 99.9739 766.92 100H727.5ZM730 88.14H764.34C765.272 88.2446 766.216 88.1351 767.1 87.82C767.34 87.62 767.46 86.68 767.46 85.01V50.77C767.46 49.1 767.34 48.17 767.1 47.96C766.216 47.6449 765.272 47.5355 764.34 47.64H730C728.33 47.64 727.4 47.75 727.19 47.96C726.98 48.17 726.88 49.1 726.88 50.77V85C726.88 86.67 726.98 87.61 727.19 87.81C727.4 88.01 728.33 88.14 730 88.14Z"
        fill="#9c27b0"
      />
      <path
        d="M790.2 100V35.78H803.13L826.41 63.52L849.6 35.78H862.6V100H850.74V52.91L826.39 81.91L802 53V100H790.2Z"
        fill="#9c27b0"
      />
      <path
        d="M886.25 100C882.97 99.9764 879.83 98.6621 877.511 96.3415C875.193 94.0208 873.881 90.8805 873.86 87.6V48.18C873.881 44.8995 875.193 41.7592 877.511 39.4385C879.83 37.1179 882.97 35.8036 886.25 35.78H925.68C928.961 35.8061 932.099 37.121 934.419 39.4408C936.739 41.7606 938.054 44.8994 938.08 48.18V87.6C938.054 90.8806 936.739 94.0194 934.419 96.3392C932.099 98.659 928.961 99.9739 925.68 100H886.25ZM888.75 88.14H923.09C924.026 88.243 924.972 88.1336 925.86 87.82C926.1 87.62 926.21 86.68 926.21 85.01V50.77C926.21 49.1 926.1 48.17 925.86 47.96C924.972 47.6464 924.026 47.5371 923.09 47.64H888.75C887.09 47.64 886.15 47.75 885.94 47.96C885.73 48.17 885.63 49.1 885.63 50.77V85C885.63 86.67 885.73 87.61 885.94 87.81C886.15 88.01 887.09 88.14 888.75 88.14Z"
        fill="#9c27b0"
      />
      <path
        d="M972 100V47.64H945.78V35.78H1010V47.64H983.87V100H972Z"
        fill="#9c27b0"
      />
      <path
        d="M1017.36 100V35.78H1029.05V100H1017.36Z"
        fill="#9c27b0"
      />
      <path
        d="M1304.36 100V35.78H1316.05V100H1304.36Z"
        fill="#9c27b0"
      />
      <path
        d="M567.43 100H553.7L527.65 54.64L501.43 100H487.78L527.65 31.28L567.43 100Z"
        fill="#9c27b0"
      />
      <path
        d="M1297.43 100H1283.7L1257.7 54.64L1231.43 100H1217.78L1257.65 31.28L1297.43 100Z"
        fill="#9c27b0"
      />
      <path
        d="M1035.81 35.78H1049.54L1075.54 81.14L1101.76 35.78H1115.41L1075.54 104.5L1035.81 35.78Z"
        fill="#9c27b0"
      />
      <path
        d="M1122.24 100V35.78H1181.37V47.64H1134.1V61.92H1172.1V73.87H1134.1V88.14H1181.37V100H1122.24Z"
        fill="#9c27b0"
      />
      <path
        d="M53.87 44.86H101.03V36.1L53.87 36.21C49.0305 36.2179 44.3917 38.1446 40.9706 41.5676C37.5494 44.9906 35.6253 49.6305 35.62 54.47V68.13H83C85.4896 68.1327 87.8764 69.1228 89.6368 70.8832C91.3972 72.6436 92.3874 75.0304 92.39 77.52V91.35H35.62V100H101V77.5C100.992 72.7221 99.0954 68.141 95.7235 64.756C92.3515 61.3709 87.7779 59.4564 83 59.43H44.29V54.43C44.2979 51.8926 45.3101 49.4615 47.1053 47.6682C48.9005 45.8749 51.3326 44.8653 53.87 44.86V44.86Z"
        fill="#fff"
      />
      <path
        d="M109.76 36.1V44.84H137.28V100H146.08V44.84H173.52V36.1H109.76Z"
        fill="#fff"
      />
      <path
        d="M258.13 91.2V36.16H249.33V100H313.09V91.2H258.13Z"
        fill="#fff"
      />
      <path
        d="M418.56 31.36L384.1 91.2H330.62V36.16H321.82V100H379.03H385.58H389.19L418.6 48.9L448.32 100.09H458.49L418.56 31.36Z"
        fill="#fff"
      />
      <path
        d="M240.59 44.84V36.1H182.26V100H240.59V91.2H191.06V68H221.76V59.3H191.06V44.84H240.59Z"
        fill="#fff"
      />
      <path
        d="M26.79 0.5C26.79 7.60976 23.9691 14.429 18.9464 19.4611C13.9238 24.4931 7.10974 27.3268 0 27.34C7.11841 27.34 13.9453 30.1678 18.9787 35.2013C24.0122 40.2347 26.84 47.0616 26.84 54.18C26.84 47.0616 29.6678 40.2347 34.7013 35.2013C39.7347 30.1678 46.5616 27.34 53.68 27.34C46.5616 27.34 39.7347 24.5122 34.7013 19.4788C29.6678 14.4453 26.84 7.61841 26.84 0.5H26.79Z"
        fill="#9c27b0"
      />
      <path
        d="M438.63 47.28V38.49H433.57V36.1H445.65V38.49H441.16V47.28H438.63Z"
        fill="#fff"
      />
      <path
        d="M445.61 47.28V36.1H449.26L451.97 40.73L454.66 36.1H458.3V47.28H455.92V38L452 44.8L448 38V47.24L445.61 47.28Z"
        fill="#fff"
      />
      <title>
        STELLA Automotive AI
      </title>
    </svg>
  </div>
  <h1
    class="MuiTypography-root MuiTypography-h1 css-1udmwja-MuiTypography-root"
  >
    Ohh, No!
  </h1>
  <h2
    class="MuiTypography-root MuiTypography-h5 css-10hv533-MuiTypography-root"
  >
    There has been an error.
  </h2>
  <h6
    class="MuiTypography-root MuiTypography-subtitle1 css-as4pw1-MuiTypography-root"
  >
    Authentication Error
  </h6>
</div>
`;

exports[`error-splash should render correctly when no error message is passed 1`] = `
<div
  class="MuiBox-root css-ui1dqj"
>
  <div
    class="MuiBox-root css-1aisqbv"
  >
    <svg
      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-3g5fjw-MuiSvgIcon-root"
      focusable="false"
      role="img"
      viewBox="0 0 1317 105"
    >
      <path
        d="M583.82 100C580.54 99.9764 577.4 98.6621 575.081 96.3415C572.763 94.0208 571.451 90.8805 571.43 87.6V35.78H583.2V85C583.2 86.67 583.3 87.61 583.51 87.81C583.72 88.01 584.65 88.13 586.32 88.13H620.66C621.592 88.2346 622.536 88.1251 623.42 87.81C623.66 87.61 623.78 86.67 623.78 85V35.78H635.64V87.6C635.614 90.8806 634.299 94.0194 631.979 96.3392C629.659 98.659 626.521 99.9739 623.24 100H583.82Z"
        fill="#9c27b0"
      />
      <path
        d="M669.53 100V47.64H643.31V35.78H707.52V47.64H681.39V100H669.53Z"
        fill="#9c27b0"
      />
      <path
        d="M727.5 100C724.22 99.9764 721.08 98.6621 718.761 96.3415C716.443 94.0208 715.131 90.8805 715.11 87.6V48.18C715.131 44.8995 716.443 41.7592 718.761 39.4385C721.08 37.1179 724.22 35.8036 727.5 35.78H766.92C770.201 35.8061 773.339 37.121 775.659 39.4408C777.979 41.7606 779.294 44.8994 779.32 48.18V87.6C779.294 90.8806 777.979 94.0194 775.659 96.3392C773.339 98.659 770.201 99.9739 766.92 100H727.5ZM730 88.14H764.34C765.272 88.2446 766.216 88.1351 767.1 87.82C767.34 87.62 767.46 86.68 767.46 85.01V50.77C767.46 49.1 767.34 48.17 767.1 47.96C766.216 47.6449 765.272 47.5355 764.34 47.64H730C728.33 47.64 727.4 47.75 727.19 47.96C726.98 48.17 726.88 49.1 726.88 50.77V85C726.88 86.67 726.98 87.61 727.19 87.81C727.4 88.01 728.33 88.14 730 88.14Z"
        fill="#9c27b0"
      />
      <path
        d="M790.2 100V35.78H803.13L826.41 63.52L849.6 35.78H862.6V100H850.74V52.91L826.39 81.91L802 53V100H790.2Z"
        fill="#9c27b0"
      />
      <path
        d="M886.25 100C882.97 99.9764 879.83 98.6621 877.511 96.3415C875.193 94.0208 873.881 90.8805 873.86 87.6V48.18C873.881 44.8995 875.193 41.7592 877.511 39.4385C879.83 37.1179 882.97 35.8036 886.25 35.78H925.68C928.961 35.8061 932.099 37.121 934.419 39.4408C936.739 41.7606 938.054 44.8994 938.08 48.18V87.6C938.054 90.8806 936.739 94.0194 934.419 96.3392C932.099 98.659 928.961 99.9739 925.68 100H886.25ZM888.75 88.14H923.09C924.026 88.243 924.972 88.1336 925.86 87.82C926.1 87.62 926.21 86.68 926.21 85.01V50.77C926.21 49.1 926.1 48.17 925.86 47.96C924.972 47.6464 924.026 47.5371 923.09 47.64H888.75C887.09 47.64 886.15 47.75 885.94 47.96C885.73 48.17 885.63 49.1 885.63 50.77V85C885.63 86.67 885.73 87.61 885.94 87.81C886.15 88.01 887.09 88.14 888.75 88.14Z"
        fill="#9c27b0"
      />
      <path
        d="M972 100V47.64H945.78V35.78H1010V47.64H983.87V100H972Z"
        fill="#9c27b0"
      />
      <path
        d="M1017.36 100V35.78H1029.05V100H1017.36Z"
        fill="#9c27b0"
      />
      <path
        d="M1304.36 100V35.78H1316.05V100H1304.36Z"
        fill="#9c27b0"
      />
      <path
        d="M567.43 100H553.7L527.65 54.64L501.43 100H487.78L527.65 31.28L567.43 100Z"
        fill="#9c27b0"
      />
      <path
        d="M1297.43 100H1283.7L1257.7 54.64L1231.43 100H1217.78L1257.65 31.28L1297.43 100Z"
        fill="#9c27b0"
      />
      <path
        d="M1035.81 35.78H1049.54L1075.54 81.14L1101.76 35.78H1115.41L1075.54 104.5L1035.81 35.78Z"
        fill="#9c27b0"
      />
      <path
        d="M1122.24 100V35.78H1181.37V47.64H1134.1V61.92H1172.1V73.87H1134.1V88.14H1181.37V100H1122.24Z"
        fill="#9c27b0"
      />
      <path
        d="M53.87 44.86H101.03V36.1L53.87 36.21C49.0305 36.2179 44.3917 38.1446 40.9706 41.5676C37.5494 44.9906 35.6253 49.6305 35.62 54.47V68.13H83C85.4896 68.1327 87.8764 69.1228 89.6368 70.8832C91.3972 72.6436 92.3874 75.0304 92.39 77.52V91.35H35.62V100H101V77.5C100.992 72.7221 99.0954 68.141 95.7235 64.756C92.3515 61.3709 87.7779 59.4564 83 59.43H44.29V54.43C44.2979 51.8926 45.3101 49.4615 47.1053 47.6682C48.9005 45.8749 51.3326 44.8653 53.87 44.86V44.86Z"
        fill="#fff"
      />
      <path
        d="M109.76 36.1V44.84H137.28V100H146.08V44.84H173.52V36.1H109.76Z"
        fill="#fff"
      />
      <path
        d="M258.13 91.2V36.16H249.33V100H313.09V91.2H258.13Z"
        fill="#fff"
      />
      <path
        d="M418.56 31.36L384.1 91.2H330.62V36.16H321.82V100H379.03H385.58H389.19L418.6 48.9L448.32 100.09H458.49L418.56 31.36Z"
        fill="#fff"
      />
      <path
        d="M240.59 44.84V36.1H182.26V100H240.59V91.2H191.06V68H221.76V59.3H191.06V44.84H240.59Z"
        fill="#fff"
      />
      <path
        d="M26.79 0.5C26.79 7.60976 23.9691 14.429 18.9464 19.4611C13.9238 24.4931 7.10974 27.3268 0 27.34C7.11841 27.34 13.9453 30.1678 18.9787 35.2013C24.0122 40.2347 26.84 47.0616 26.84 54.18C26.84 47.0616 29.6678 40.2347 34.7013 35.2013C39.7347 30.1678 46.5616 27.34 53.68 27.34C46.5616 27.34 39.7347 24.5122 34.7013 19.4788C29.6678 14.4453 26.84 7.61841 26.84 0.5H26.79Z"
        fill="#9c27b0"
      />
      <path
        d="M438.63 47.28V38.49H433.57V36.1H445.65V38.49H441.16V47.28H438.63Z"
        fill="#fff"
      />
      <path
        d="M445.61 47.28V36.1H449.26L451.97 40.73L454.66 36.1H458.3V47.28H455.92V38L452 44.8L448 38V47.24L445.61 47.28Z"
        fill="#fff"
      />
      <title>
        STELLA Automotive AI
      </title>
    </svg>
  </div>
  <h1
    class="MuiTypography-root MuiTypography-h1 css-1udmwja-MuiTypography-root"
  >
    Ohh, No!
  </h1>
  <h2
    class="MuiTypography-root MuiTypography-h5 css-10hv533-MuiTypography-root"
  >
    There has been an error.
  </h2>
</div>
`;
